packages/legend-studio/src/stores/LegendStudioPlugin.ts

```diff
@@ -68,10 +69,20 @@ export type ModelLoaderExtensionConfiguration = {
   renderer: (editorStore: EditorStore) => React.ReactNode | undefined;
 };
 
-export type TestableMetadataGetter = (
+export type TestableElementEditorOpener = (
```

Comment:
Don't think we need this separate type if we don't intend to use it, we can go light by inlining it